gems/aws-sdk-core/lib/aws-sdk-core/plugins/retries/retry_quota.rb

```diff
@@ -0,0 +1,54 @@
+module Aws
+  module Plugins
+    module Retries
+
+      # @api private
+      # Used in 'standard' and 'adaptive' retry modes.
+      class RetryQuota
+        INITIAL_RETRY_TOKENS = 500
+        RETRY_COST = 5
+        NO_RETRY_INCREMENT = 1
+        TIMEOUT_RETRY_COST = 10
+
+        def initialize
```

Comment:
Let's configure max capacity, and set available capacity to it. Default to initial